README.md

```diff
@@ -25,8 +25,7 @@ I gave at dotScale
 (or [read the slides](https://speakerdeck.com/fabpot/a-monorepo-vs-manyrepos))...
 or watch the longer version from
 [DrupalCon](https://www.youtube.com/watch?v=4w3-f6Xhvu8).
-["The Monorepo - Storing your source code has never been so much fun"](The Monorepo - Storing your source code has never been so much fun)
-is also a great resource.
+"[The Monorepo - Storing your source code has never been so much fun](https://www.oreilly.com/library/view/velocity-conference-2017/9781491985335/video316236.html)" is also a great resource.
```

Comment:
Apparently, that's only the first minute of it. If that's the case, we should remove the link instead.

Reply:
I see. 
Will remove the link.